### PR TITLE
feat: enable send for btc snap

### PR DESCRIPTION
**Pull request by montelaidev**
## **Description**

This PR enables the send feature for `@metamask/bitcoin-wallet-snap`

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/564

## **Manual testing steps**

1. Create a btc account
2. Switch to the btc account
3. Click send 

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by montelaidev** [comment]
@metamaskbot update-policies`

**Comment by montelaidev** [comment]
@SocketSecurity ignore npm/@metamask/bitcoin-wallet-snap@0.8.0

**Comment by ccharly** [comment]
https://github.com/SocketSecurity ignore npm/@metamask/bitcoin-wallet-snap@0.8.1

This is our own Snap.

File: ui/components/app/wallet-overview/btc-overview.test.tsx [hunk 1]
```diff
@@ -161,11 +160,7 @@ describe('BtcOverview', () => {
   it('buttons Send/Swap/Bridge are disabled', () => {
```
Comment (plasmacorral) [inline ui/components/app/wallet-overview/btc-overview.test.tsx:160]: We should remove "Send" from this line

**Comment by montelaidev** [comment]
This PR is replaced by this new [PR](https://github.com/MetaMask/metamask-extension/pull/27963)

**Pull request closed without merging**